Robert Khromov/src/app/pages/User.js

```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react';
+import UserProfile from '../components/User';
+import {connect} from "react-redux";
+import {getUsers} from "../actions/user";
+
+class User extends Component {
+    componentDidMount() {
+        getUsers();
```

Comment:
```suggestion
        this.props.dispatch(getUsers());
```
благодаря функции `connect()` нам доступен метод `dispatch()`